catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/transformer/receive/SQSMessageReceiverTest.scala

```diff
@@ -72,7 +72,12 @@ class SQSMessageReceiverTest
   )(testWith: TestWith[SQSMessageReceiver, R]) = {
 
     val snsWriter =
-      maybeSnsWriter.getOrElse(new SNSWriter(snsClient, SNSConfig(topic.arn)))
+      maybeSnsWriter.getOrElse(
+        new SNSWriter(
+          snsClient,
+          SNSConfig(topic.arn),
+          s3Client,
```

Comment:
You can probably just inject: https://github.com/wellcometrust/platform/blob/master/common/src/main/scala/uk/ac/wellcome/s3/S3ObjectStore.scala

Reply:
👍 Presumed this was implicit from previous comments.